desktop/src/components/custom/mock-video.tsx

```diff
@@ -85,12 +87,12 @@ export const MockVideo: React.FC<MockVideoProps> = ({
       x: WIDTH/2-WIDTH/4, // default: 0
       y: HEIGHT-200, // default: 0
       width: WIDTH/2, // default: null (full width)
-      height: 120, // default: null (full height)
+      height: parseInt(subtitleStyle) + 20, // default: null (full height)

```

Comment:
I think we should use an actual  `subtitle layer` see [desktop/src/lib/subtitle-layer.ts](https://github.com/acowusu/segp/blob/main/desktop/src/lib/subtitle-layer.ts)

This is to ensure the preview actually matches the generated video